ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -104,7 +104,7 @@ object LedgerApiTestTool {
     }
 
     val allTestCaseNames: Set[String] =
-      (Tests.all ++ Tests.retired).flatMap(_.tests).map(_.name).toSet
+      (Tests.all(config) ++ Tests.retired).flatMap(_.tests).map(_.name).toSet
```

Comment:
You're recreating over and over again all the tests suites across the entire class.

Please create a variable, call `Test.all(config)` once and use that value.

Reply:
addressed comment